themes/bootstrap3/templates/Recommend/ConsortialVuFind.phtml

```diff
@@ -0,0 +1,31 @@
+<div class="consortialVuFind">
+  <?php $results = $this->recommend->getResults() ?>
+  <?php $records = $results['records'] ?? [] ?>
+  <?php $resultCount = $results['resultCount'] ?? 0; ?>
+  <?php $moreResultsUrl = $this->recommend->getMoreResultsUrl(); ?>
+  <?php if (!empty($records)): ?>
+    <h2><?=$this->transEsc('consortial_vufind_recommend_heading')?></h2>
+    <?php if ($this->translate('consortial_vufind_recommend_intro', [], '')): ?>
+      <p><?=$this->translate('consortial_vufind_recommend_intro', [
+          '%%result_count%%' => $this->localizedNumber($resultCount),
+          '%%url%%' => $moreResultsUrl,
+      ])?></p>
+    <?php endif; ?>
+    <div class="list-group">
```

Comment:
Since this is a list, would it be more appropriate to use ul/li tags? I imagine this is modeled on some other precedent in the code, but since recent accessibility work has introduced more semantic tags in more places, I'm wondering if we should consider that here as well.

Reply:
Yes 100% thank you for the catch, fixed.  This was another copy/paste I think, but it's definitely an anti-pattern.  And it looks like I did the same in EPF results...I'm going to look for more of those as a separate PR.